src/Mobs/Cow.h

```diff
@@ -20,7 +20,10 @@ class cCow :
 	virtual void GetDrops(cItems & a_Drops, cEntity * a_Killer = nullptr) override;
 	virtual void OnRightClicked(cPlayer & a_Player) override;
 
-	virtual const cItem GetFollowedItem(void) const override { return cItem(E_ITEM_WHEAT); }
+	virtual void GetFollowedItems(cItems & a_Items) override
```

Comment:
Something like this:
```cpp
virtual cItems GetFollowedItems() override
{
	cItems FollowedItems;
	FollowedItems.Add(E_ITEM_MY_ITEM1);
	FollowedItems.Add(E_ITEM_MY_ITEM2);
	FollowedItems.Add(E_ITEM_MY_ITEM3);
	return FollowedItems;
}
```

But you might want to wait with it until more people agree.